src/routes/search/components/SearchActiveHeader.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components/native'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
+import HeaderBackButton from 'react-navigation-stack/dist/views/Header/HeaderBackButton'
+import { SearchBar } from 'react-native-elements'
+
+const ThemedSearchBar = styled(SearchBar).attrs(props => ({
+  containerStyle: {
+    flexGrow: 1,
+    backgroundColor: props.theme.colors.backgroundAccentColor,
+    borderTopColor: props.theme.colors.backgroundAccentColor,
+    borderBottomColor: props.theme.colors.backgroundAccentColor
+  },
+  inputContainerStyle: {
+    backgroundColor: props.theme.colors.backgroundColor
+  },
+  inputStyle: {
+    backgroundColor: props.theme.colors.backgroundColor
+  }
+}))``
+
+const HorizontalLeft = styled.View`
+  flex:1;
+  flex-direction: row;
+  align-items: center;
+`
+
+const BoxShadow = styled.View`
+  elevation: 1;
+  background-color: ${props => props.theme.colors.backgroundAccentColor};
+  height: ${props => props.theme.dimensions.headerHeight};
+`
+
+type PropsType = {|
+  theme: ThemeType,
+  query: string,
+  closeSearchBar: () => void,
+  onSearchChanged: (query: string) => void
+|}
+
+class SearchActiveHeader extends React.Component<PropsType> {
+  render () {
```

Comment:
Could you try these changes for enabling auto-focus?

(according to https://react-native-training.github.io/react-native-elements/docs/searchbar.html#a-name-calling-a-calling-methods-on-searchbar this should work)

```suggestion
  componentDidMount () {
    this.search.focus()
  }

  render () {
```

Reply:
In fact I tried, but it didn't work, so I created a task. So I would do it in a separate task.